test/SlowTests/Issues/RavenDB-15909.cs

```diff
@@ -47,5 +50,44 @@ public async Task Cached_Properties_Should_Renew_Correctly()
                 }
             }
         }
+
+        [Fact]
+        public async Task Cached_Properties_Should_Renew_Correctly_Big_Stream()
+        {
+            using (var store = GetDocumentStore())
+            {
+                const int documentsCount = 8;
+                using (var bulkInsert = store.BulkInsert())
+                {
+                    for (var i = 0; i < documentsCount; i++)
+                    {
+                        var expandoObject = new ExpandoObject();
+                        for (var j = 0; j < CachedProperties.CachedPropertiesSize; j++)
+                        {
+                            expandoObject.TryAdd((i + j).ToString(), null);
+                        }
+
+                        await bulkInsert.StoreAsync(expandoObject, i.ToString());
+                        expandoObject.Remove("Id", out _);
```

Comment:
Why do we call `Remove` here?

Reply:
Currently there is no need to do that. The previous code was reusing the same object.